src/main/java/duke/controller/Duke.java

```diff
@@ -0,0 +1,37 @@
+package duke.controller;
+
+import duke.exception.InvalidInputException;
+import duke.ui.*;
```

Comment:
Imported classes should always be listed explicitly. Could refer to coding standards

Reply:
Done! Thank you!